.github/workflows/publish_lib_to_npm.yml

```diff
@@ -0,0 +1,43 @@
+name: Publish lib/ (and convenience package autometrics/) packages to NPMJS
+on:
+  push:
+    tags:
+      - "lib-*"
+jobs:
+  test:
```

Comment:
Is this here as a requirement for the `publish` job? If so, then we should probably add the `needs: test` property to the `publish` job. (see: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds)

Reply:
ah nice one! Updating now